sdk/python/arvados/keep.py

```diff
@@ -22,6 +22,16 @@
 global_client_object = None
 
 
+# Monkey patch TCP constants when not available (apple). Values sourced from:
+# http://www.opensource.apple.com/source/xnu/xnu-2422.115.4/bsd/netinet/tcp.h
+if not hasattr(socket, 'TCP_KEEPALIVE'):
+    socket.TCP_KEEPALIVE = 0x010
+if not hasattr(socket, 'TCP_KEEPINTVL'):
+    socket.TCP_KEEPINTVL = 0x101
+if not hasattr(socket, 'TCP_KEEPCNT'):
+    socket.TCP_KEEPCNT = 0x102
```

Comment:
Do those Apple values succeed in Windows? I don't think we can expect the Apple values to work on all platforms where Python doesn't define them for us. E.g., `TCP_KEEPINTVL` is 0x5 in linux, 0x200 in FreeBSD... Setting the wrong sockopts seems like a recipe for confusion, at best.